test/test.compute.ts

```diff
@@ -66,6 +66,19 @@ describe('Compute auth client', () => {
     });
   });
 
+  it('should refresh if access token has expired', (done) => {
```

Comment:
it would good to add another check exercises the other path – namely these case where`refreshTokenEarlyMillis` has been specified but the token should not be expired.

Reply:
Done.